### DISCUSSION TITLE
How can I just open kitty on neovim without any bs ?

### DISCUSSION THREAD

**Discussion by koaydemir** [mikesmithgh/kitty-scrollback.nvim#268] [General]
So, I don't want to use kitty scrollback, because i used it before expecting it to just open neovim, or at least just open vim and put the text from my terminal inside it. It turned out to be way more complicated and I didn't want to learn kitty-scrollback stuff just to use an editor i've been using for 2 years. The popups it shows and stuff just, doesn't work. I never read the docs though.

So that's why I don't wanna use it but I want to learn if there's a way to just: send the text on my terminal right into a new neovim buffer. Nothing else.

**Comment by mikesmithgh** [comment]
> without any bs ? 

> I never read the docs though.

![heavy-breathing](https://github.com/user-attachments/assets/565e5eb9-1a72-44c1-88ad-8d96c9fe7864)

> So, I don't want to use kitty scrollback

Check out https://github.com/mikesmithgh/kitty-scrollback.nvim?tab=readme-ov-file#-alternatives for alternatives that may be better for your use case.
